alerting-config-service-api/src/main/proto/org/hypertrace/alerting/config/service/v1/metric_anomaly_event_condition.proto

```diff
@@ -0,0 +1,103 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package org.hypertrace.alerting.config.service.v1;
+
+message MetricAnomalyEventCondition {
+  MetricSelection metric_selection = 1;
+  repeated ViolationCondition violation_condition = 2;
+  string rule_duration = 3; // iso duration string
```

Comment:
I have this question each time, so maybe a naming clarification is in order - what is this duration vs the interval on the selection below? Is this how long a violation must be active before it's considered an anomaly?

Reply:
hmm - maybe something like `lookback_duration` or `evaluation_window_duration` ?